tests/conftest.py

```diff
@@ -325,3 +326,22 @@ def VMEC_save(SOLOVEV, tmpdir_factory):
     )
     desc = Dataset(str(SOLOVEV["desc_nc_path"]), mode="r")
     return vmec, desc
+
+
+@pytest.fixture(scope="session")
+def quadratic_flux_equilibriums(tmpdir_factory):
+    """Create equilibriums for quadratic flux tests."""
+    surface = FourierRZToroidalSurface(
+        R_lmn=[1, 0.125, 0.1],
+        Z_lmn=[-0.125, -0.1],
+        modes_R=[[0, 0], [1, 0], [0, 1]],
+        modes_Z=[[-1, 0], [0, -1]],
+        NFP=4,
+    )
+    nonaxisym_eq_with_surface = Equilibrium(L=4, M=4, N=4, surface=surface)
+    nonaxisym_eq_with_surface.solve(verbose=0)
```

Comment:
would be better to compute these once and store them in `tests/inputs` rather than recomputing on each test run